app/components/work_package_relations_tab/add_work_package_child_form_component.html.erb

```diff
@@ -16,6 +16,11 @@
           end
         end
         flex.with_row do
+          # @workpackage is not available inside the render_inline_form block
+          # so we need to re-define them here. Figure out solution for this.
+          ancestors_id = @work_package.ancestors.pluck(:id)
+          children_id = @work_package.children.pluck(:id)
+          non_relatable_wp_ids = [@work_package.id] + ancestors_id + children_id
```

Comment:
We have already an endpoint called `available_relation_candidates` to find all allowed values for a relation which you can simply use. Please have a look at [how it is done for the "normal" relations](https://github.com/opf/openproject/blob/dev/app/components/work_package_relations_tab/work_package_relation_form_component.html.erb#L23) and just do the same thing here.